Dockerfile-ubuntu2204-icpx

```diff
@@ -22,7 +22,8 @@ RUN apt-get update && apt-get -y install \
   wget \
   xz-utils \
   net-tools \
-  tcpreplay
+  tcpreplay \
+  llvm
```

Comment:
nitpick: should be sorted.

Reply:
Only moved the llvm because all of other dockerfiles uses same order (tcpreplay and nettools at the end)